sigar_windows_test.go

```diff
@@ -95,4 +96,78 @@ var _ = Describe("SigarWindows", func() {
 			Ω(sd.Description).Should(Equal("Windows"))
 		})
 	})
+
+	Describe("ProcessList", func() {
+		It("gets the list of processes", func() {
+			pl := sigar.ProcList{}
+			err := pl.Get()
+			Ω(err).ShouldNot(HaveOccurred())
+			Ω(len(pl.List)).Should(BeNumerically(">", 0))
+		})
+	})
+
+	Describe("ProcessState", func() {
+		It("gets the process detail", func() {
+			pl := sigar.ProcList{}
+			err := pl.Get()	
+			Ω(err).ShouldNot(HaveOccurred())
+			for _, pid := range pl.List {
+				details := sigar.ProcState{}
+				err  := details.Get(pid)
+				fmt.Printf("Pid: %v Name: %v Err: %v\n", pid, details.Name, err)
```

Comment:
Is the idea that we should manually verify the provided process list and details? I think its okay as-is but would be nice to compare the output against the `tasklist` command or similar.